client/index.js

```diff
@@ -64,6 +82,9 @@ const sox = (args = {}) => {
   // send : String -> a -> Async Action
   socket.send = send
 
+  // send : Object -> String -> a -> Async Action
```

Comment:
In the past we've used `{ k: v }` in signatures, rather than `Object`.  Following ramda a bit on that, I think.